dev/NavigationView/NavigationViewItemsFactory.h

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "common.h"
+#include "ElementFactory.h"
+
+class NavigationViewItemsFactory :
+    public winrt::implements<NavigationViewItemsFactory, ElementFactory>
+{
+public:
+    NavigationViewItemsFactory();
+
+    void UserElementFactory(winrt::Windows::UI::Xaml::IElementFactory elementFactory) { m_userElementFactory = elementFactory; };
+
+#pragma region IElementFactoryOverrides
+    winrt::UIElement GetElementCore(winrt::ElementFactoryGetArgs const& args);
+    void RecycleElementCore(winrt::ElementFactoryRecycleArgs const& args);
+#pragma endregion
+
+private:
+    winrt::Windows::UI::Xaml::IElementFactory m_userElementFactory{ nullptr };
+    winrt::Windows::UI::Xaml::ElementFactoryGetArgs m_getArgsWUX{};
```

Comment:
>m_getArgsWUX [](start = 52, length = 12)

Can these args be static? We set all the relevant properties just before using them.